force-app/main/default/classes/SOQL.cls

```diff
@@ -117,7 +135,9 @@ public with sharing class SOQL {
      * @return A list of Case records.
      */
     public static List<Case> getFilterCaseOrigins() {
-        return null; // Replace null with the variable you used to store the result
+        
+        Case[] caseList = [SELECT CaseNumber, Origin, Status FROM Case WHERE NOT (Origin IN ('Web', 'Email', null)) WITH SECURITY_ENFORCED ];
```

Comment:
This is actually closer to my solution/the best way to do it for query selectivity. You end up running this query against a table with a lot of records, and you want to avoid using the NOT operator. Instead, you want to specify what you DO want... the downside to that is that you have to always know what values the picklist value set contains. So, it can hurt readability sometimes at the cost of being safe for scaling. Something to consider.

You could also rephrase this as WHERE Origin NOT IN (...)